docs/host_for_multiuser/setup.md

```diff
@@ -64,88 +66,95 @@ Follow the steps below to setup `multiuser` mode.
 8. Click "Generate new private key" in "Firebase Admin SDK" section.
 9. Put the downloaded file to `studio/config/auth/firebase_private.json`.
```

Comment:
Save the downloaded file to
or 
Put the downloaded file in

Reply:
Corrected point above.